src/Request.php

```diff
@@ -1064,8 +1064,8 @@ private static function limitTelegramRequests($action, array $data = [])
                     $requests = DB::getTelegramRequestCount($chat_id, $inline_message_id);
 
                     if ($requests['LIMIT_PER_SEC'] == 0     // No more than one message per second inside a particular chat
```

Comment:
I think it would make this query easier to understand, if each part was saved in a separate variable before the `if`.